src/analysis/openstreetmap/qgsosmdatabase.h

```diff
@@ -82,7 +82,6 @@ class ANALYSIS_EXPORT QgsOSMDatabase
 
     QgsOSMTags tags( bool way, QgsOSMId id ) const;
 
-    //! \note available in Python bindings
```

Comment:
would be worth adding a comment here
`SIP does not seem to handle this return type - strange`